src/constraints/electric_utility_constraints.jl

```diff
@@ -52,7 +52,7 @@ function add_export_constraints(m, p; _n="")
             @warn "Adding binary variable for net metering choice. Some solvers are slow with binaries."
 
             # Good to bound the benefit - we use max_bene as a lower bound because the benefit is treated as a negative cost
-            max_bene = sum([ld*rate for (ld,rate) in zip(p.s.electric_load.loads_kw, p.s.electric_tariff.export_rates[:NEM])])*10
+            max_bene = sum([ld*rate for (ld,rate) in zip(p.s.electric_load.loads_kw, p.s.electric_tariff.export_rates[:NEM])])*p.pwf_e*p.hours_per_time_step*10
```

Comment:
Just curious why `*p.hours_per_time_step` helps here?

Reply:
Alex, this bound on NEM is less important than the WHL bound, because the NEM benefit also gets constrained by net metering size limit and the constraint that the system cannot produce > 100% of the site's load, right? So, wouldn't the NEM benefit never be > `[ld*rate for (ld,rate) in zip(p.s.electric_load.loads_kw, p.s.electric_tariff.export_rates[:NEM])]` ?

Reply:
Oh! But this limit is also used for the EXC bin, so if someone supplied a very high `export_rate_beyond_net_metering_limit` this may become active. 

Let me know if this thinking is off.

Reply:
Thats correct on the wholesale and excess, you still need a large enough limit for the excess in particular.  Also, the net metering is bounded elsewhere but that bound was too low due to the present worth factor being >10.  Including `p.hours_per_time_step` and, more importantly, the annuity factor `p.pwf_e` just makes the big-M reconcile with the other components of the benefit calculation: https://github.com/NREL/REopt.jl/blob/209ba633c5601b4fdb0d417e9019e6616ebb42b8/src/constraints/electric_utility_constraints.jl#L89-L91

Reply:
Thanks for clarifying!